src/Blazored.Typeahead/Forms/BlazoredTypeaheadInput.razor

```diff
@@ -5,12 +5,14 @@
 <div class="blazored-typeahead" @ref="typeahead">
     <div class="blazored-typeahead__controls">
         <input @ref="searchInput"
-               class="blazored-typeahead__input @CssClass @(!IsShowingSearchbar ? "blazored-typeahead__input-hidden" : null)"
                @bind-value="@SearchText"
                @bind-value:event="oninput"
                type="text"
                autocomplete="off"
-               placeholder="@Placeholder" />
+               placeholder="@Placeholder"
+               @attributes="@Attributes"

```

Comment:
This can be changed to use the `AdditionalAttributes` property provided by `InputBase`
```suggestion
               @attributes="@AdditionalAttributes"
```